python-api/scripts/dbusConnection3.py

```diff
@@ -17,7 +17,7 @@
 #print(type(mode), mode)
 # mode=0
 # while mode<=3:
-    # print("mode:", mode)
+#     print("mode:", mode)
```

Comment:
I'd love to delete the whole scripts folder as it is completely outdated. will do so.